lib/ui/utilities/validation.dart

```diff
@@ -12,12 +11,11 @@ class Validation {
     }
   }
 
-    static String? validateField(String? s, {String? errorMessage}) {
+  static String? validateField(String? s, {String? errorMessage}) {
     if (StringUtil.isEmpty(s)) {
       return errorMessage ?? "Required";
```

Comment:
kindly help with the required as well

Reply:
I should remove the question mark?

Reply:
nope, add the required to the intl as well

Reply:
this is still hanging...

Reply:
okay